docs/Diagnostics.md

```diff
@@ -0,0 +1,82 @@
+This document contains information about diagnostic tools and outputs used for debugging and diagnosing eBPF issues.
```

Comment:
I'm not seeing any document in this PR that contains a link to these documents.  These need to be discoverable via some path from the main README.  For example, debugging.md is linked to from tutorial.md.

Please update some existing docs to add pointers to these new docs.

Reply:
Added a line in the readme to direct to the TSG.